src/command_manager.py

```diff
@@ -35,7 +37,14 @@ def get_control_parameter(self, pgdata, parameter, preproc=None, log=True):
         res = helpers.subprocess_popen(command, log_cmd=log)
         if not res:
             return None
-        value = res.communicate()[0].decode('utf-8').split(':')[-1].strip()
+        (stdout, stderr) = res.communicate()
+        if stderr:
```

Comment:
Something in stderr does not always indicate an error, it can be a warning.
I think, we should analyze `res.returncode` to detect error condition here.